chia/cmds/wallet_funcs.py

```diff
@@ -308,21 +310,37 @@ def timestamp_to_time(timestamp):
     return datetime.fromtimestamp(timestamp).strftime("%Y-%m-%d %H:%M:%S")
 
 
-async def print_offer_summary(wallet_client: WalletRpcClient, sum_dict: dict):
+async def print_offer_summary(cat_name_resolver: CATNameResolver, sum_dict: Dict[str, int], has_fee: bool = False):
```

Comment:
passing in `cat_name_resolver` instead of `wallet_client` makes it easier to write a unit test to test this function.